### DISCUSSION TITLE
Linux .clap porting of this sampler?

### DISCUSSION THREAD

**Discussion by ghost** [BOBONA/Just-a-Sample#3] [Ideas]
Hi,
I like this project! The GUI and the concept behind it is awesome! Have in the plan to port it in the Linux world? It would be fantastic! Thank you,

Regards

**Comment by BOBONA** [comment]
Since this plugin is built with JUCE, clap is not an option. But I can play around with compiling VST3 on Linux. No promises though :)

**Comment by BOBONA** [comment]
I've added a Linux VST3 build on the releases page. It seems to work fine on my Ubuntu VM. If you run into any issues, please let me know.

**Comment by ghost** [comment]
Thank you very much! I'll definitely try it in the next few days! Thank you very much, if I can, I advertise the Linux version in the various forums.....  The .clap format interested me not only because of its potential, but also because I use bitwig. Thanks anyway for the .vst3 format

I found this beautiful sampler on this youtube channel:
https://www.youtube.com/watch?v=JX-O2LBpenQ

Regards

**Comment by ghost** [comment]
Hi,
Just to tell you that to date there have been no problems with your sampler in Reaper 7.30. Thank you very much

https://i.imgur.com/92J9Csm.png

regards
